app/jobs/email_article_recruit_job.rb

```diff
@@ -22,8 +22,8 @@ def perform(publication:, current_user: nil, previous_event: nil)
     emails = [system_email.split(',')].flatten
     emails << current_user[:email] if current_user
     emails << publication.author_publications.map(&:email)
```

Comment:
add `if Rails.env.production?` so this only gets added for production.. remove the following comment